node/network/availability-distribution/src/lib.rs

```diff
@@ -135,15 +138,24 @@ impl AvailabilityDistributionSubsystem {
 			};
 			match message {
 				FromOrchestra::Signal(OverseerSignal::ActiveLeaves(update)) => {
+					let cloned_leaf = match update.activated.clone() {
```

Comment:
Why not use the span provided by overseer ?

Reply:
In this case the borrow-checker was not letting me use the `update`, so had to clone it.